curriculum/js1/4.js

```diff
@@ -6,8 +6,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, result="") => {
+  if (a === 0) return result;
+  result += b;
+  a -= 1;
+  return solution(a, b, result);
```

Comment:
you don't need to do an extra step of `a -= 1` AND `result += b`. simply `return solution( a-1, result + b)`